port/posix/posix_transport_shm.h

```diff
@@ -20,6 +20,31 @@
  * port/posix_transport_shm.h
  *
  * wolfHSM Transport Mem binding using POSIX shared memory functionality
+ *
+ * For this implementation, the server creates a POSIX shared memory object
+ * named config->name of a size to hold a common header block, the request and
+ * response buffers, and an optional DMA section.  Note POSIX specifies the
+ * name should start with "/", include no other "/" characters, and be less than
+ * NAME_MAX (limits.h) long.
+ *
+ * Once the server creates the named shared memory object, it sets the full
+ * size, maps it, and updates the header block with the configured sizes. After
+ * completing updates to the header block, it then builds a TransportMem
+ * server context using the request and response buffers.  Note the server will
+ * attempt to unlink the named shared memory object prior to creation.
+ *
+ * The client is configured with only the name of the shared object and it busy-
+ * retries to open the named shared object.  It then maps the header block,
+ * reads the configuration sizes, and configures a TransportMem client context.
+ * The client also unlinks the shared object on successful mapping.
```

Comment:
This behavior is only present in the `#if 0` code in the client init. See below comments for more

Reply:
Yup.  This is incorrect.  Updated.